src/java/main/org/apache/zookeeper/client/FourLetterWordMain.java

```diff
@@ -86,7 +86,7 @@ public static String send4LetterWord(String host, int port, String cmd, boolean
             throws IOException, SSLContextException {
         LOG.info("connecting to {} {}", host, port);
         Socket sock;
-        InetSocketAddress hostaddress= host != null ? new InetSocketAddress(host, port) :
+        InetSocketAddress hostaddress = host != null ? new InetSocketAddress(host, port) :
```

Comment:
it seems that this PR includes some code formats and typos which isn't related.

Reply:
Agreed, it's not ideal, but size of the patch makes it easy to deal with I would say.

Reply:
This is generally due to some auto format done in editor.

Reply:
@mjeelanimsft generally, this is due to the auto-format in the editor, but I think we should remove this change from this diff, given we haven't changed other logic except this one.